third-party/realdds/include/realdds/dds-stream-impl.h

```diff
@@ -0,0 +1,42 @@
+// License: Apache 2.0. See LICENSE file in root directory.
```

Comment:
This file should be in src/
I already moved the device-impl.h there, too (it was left in include/ by mistake)

Reply:
Done